ros_buildfarm/config/source_build_file.py

```diff
@@ -92,6 +92,11 @@ def __init__(self, name, data):
             self.skip_ignored_repositories = \
                 bool(data['skip_ignored_repositories'])
 
+        if '_config' in data['targets']:
+            if 'custom_rosdep_urls' in data['targets']['_config']:
+                self.custom_rosdep_urls = data['targets']['_config']['custom_rosdep_urls']
```

Comment:
Please always add the member variable to this class. E.g. with a default value of an empty list.

Then you also don't have to use `hasattr` above but can unconditionally access the member.